### PR TITLE
add note about miniforge to introduction

### PR DESCRIPTION
**Pull request by ilanschnell**
Adding a few sentences about miniforge to the introduction, including download link.

### CODE REVIEW

**Comment by scopatz** [comment]
Thanks @ilanschnell - I also think there needs to be a note about miniforge on the landing page (the index.html file)

**Comment by jjhelmus** [comment]
If miniforge is going to be promoted further could a location for issue report be created and mentioned in the same location?

**Comment by scopatz** [comment]
Yep, that is a great suggestion @jjhelmus!

File: src/user/introduction.rst [hunk 1]
```diff
@@ -71,4 +71,14 @@ From now on using ``conda install <package-name>`` will also find packages in ou
...
   
+.. note::
+
+  `Miniforge <https://github.com/conda-forge/miniforge>`__ is a community
+  effort to provide Miniconda-like installers, with the added feature that
+  conda-forge is the default channel.
+  For certain platforms (such as ARMv8 64-bit, formally known as `aarch64`)
+  Anaconda, Inc. does not provide any Miniconda installers, however Miniforge
+  does.
+  Miniforge installers are available here: https://github.com/conda-forge/miniforge/releases
```
Comment (isuruf) [inline src/user/introduction.rst]:
```suggestion
  Miniforge installers are available here: https://github.com/conda-forge/miniforge/#download
```

**Comment by ilanschnell** [comment]
I agree with @jjhelmus miniforge issues should also go into https://github.com/conda-forge/miniforge/issues.  @scopatz I've added a paragraph to the main index.html as well.

**Comment by scopatz** [comment]
Can you update the link too please @ilanschnell? Thanks!

File: src/user/introduction.rst (hunk 1 above)
Comment (scopatz) [reply src/user/introduction.rst]: This link needs to be updated too

**Pull request merged**